src/copytrade/copytrade.es6

```diff
@@ -367,6 +358,40 @@ const initConfigWindow = () => {
   win.dialog( 'open' );
 };
 
+const getTraderList = () => {
+  liveapi
+  .cached
+  .send({
+    copytrading_list : 1
+  }).then(list => {
+    const traderTokens = list.copytrading_list.traders;
+    if (!traderTokens.length) {state.is_loading = false;}
+    for (let traderToken of traderTokens) {
+      try {
```

Comment:
Could we put the `try/catch` block just before the `for` loop?

Reply:
since we don't want to continue the loop even if an exception occurs during one iteration, the exception handler should be outside the loop, you're right.